test/dlc_tests/ec2/pytorch/inference/test_pytorch_inference.py

```diff
@@ -282,6 +282,7 @@ def ec2_pytorch_inference(image_uri, processor, ec2_connection, region):
 
     inference_cmd = test_utils.get_inference_run_command(image_uri, model_name, processor)
     docker_runtime = "--runtime=nvidia --gpus all" if "gpu" in image_uri else ""
+    docker_init = " --init" if "ec2" in image_uri else ""
```

Comment:
nit: you can use `test_utils.is_ec2_image(image_uri)` instead of looking for substring directly